falafel/qe.py

```diff
@@ -399,38 +388,25 @@ def qe_shear(px,mlmax,Talm=None,fTalm=None):
     shear_alm=ttalmsp2+ttalmsm2
     return shear_alm
 
-def qe_pointsources(px,theory_func,theory_crossfunc,mlmax,fTalm=None,fEalm=None,fBalm=None,estimators=['TT','TE','EE','EB','TB','mv','mvpol'],xfTalm=None,xfEalm=None,xfBalm=None):
+def qe_pointsources(px,mlmax,fTalm,xfTalm=None,):
     """
     Inputs are Cinv filtered alms.
     px is a pixelization object, initialized like this:
     px = pixelization(shape=shape,wcs=wcs) # for CAR
     px = pixelization(nside=nside) # for healpix
     output: Point source estimator alms
     """
-    ests = estimators
-    ells = np.arange(mlmax)
-    th = lambda x,y: theory_func(x,y)
-    th_cross=lambda x,y: theory_crossfunc(x,y)
-    kfunc = lambda x: deflection_map_to_kappa_curl_alms(px,x,mlmax)
-    omap = enmap.zeros((2,)+px.shape,px.wcs) #load empty map with SO map wcs and shape
-
     if xfTalm is None:
-        if fTalm is not None: xfTalm = fTalm.copy()
-    if xfEalm is None:
-        if fEalm is not None: xfEalm = fEalm.copy()
-    if xfBalm is None:
-        if fBalm is not None: xfBalm = fBalm.copy()
+        xfTalm = fTalm.copy()
+    omap = enmap.zeros((2,)+px.shape,px.wcs) #load empty map with SO map wcs and shape
     rmap=px.alm2map_spin(np.stack((fTalm,fTalm)),0,0,ncomp=2,mlmax=mlmax)
```

Comment:
Is this really needed? Isn't the source estimator just a simple spin 0
map2alm( alm2map(alm_inverse_filtered_1) * alm2map(alm_inverse_filtered_2) ) ?

Reply:
That's right, I should have mentioned that I am working with the `bias_harden` branch and the two changes that you mentioned above were already there but I didn't update the pull request.

Reply:
Oh cool! In the future, let's try to get things merged in (or at least made aware through a PR) as soon as possible.